frontend/src/components/administration/users/users-list.tsx

```diff
@@ -0,0 +1,106 @@
+import { FC, useState } from 'react';
+
+import { User, Role } from '@autoline/shared/common/types/types';
+import EditIcon from '@mui/icons-material/Edit';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import { useUpdateUserMutation } from '@store/queries/users';
+
+import { DialogEditUser } from './dialog-edit-user';
+
+type UsersListProps = {
+  users: User[];
+};
+
+const UsersList: FC<UsersListProps> = ({ users }) => {
+  const [open, setOpen] = useState(false);
```

Comment:
`open` isn't descriptive enough in the context of the list component. "Who is opened? Is the list opened?"

Reply:
That's true, thanks. Renamed it to `openedUserEditing`.